test/Microsoft.Azure.WebJobs.Host.EndToEndTests/ServiceBusEndToEndTests.cs

```diff
@@ -449,6 +473,26 @@ public class ServiceBusTestJobs : ServiceBusTestJobsBase
             {
                 output = input;
             }
+
+            [NoAutomaticTrigger]
+            public static async Task ServiceBusBinderTest(
+                string message,
+                int numMessages,
+                Binder binder)
+            {
+                var attribute = new ServiceBusAttribute(BinderQueueName)
+                {
+                    EntityType = EntityType.Queue
+                };
+                var collector = await binder.BindAsync<IAsyncCollector<string>>(attribute);
```

Comment:
Yeah, this test doesn't verify caching, but exercises the full end to end. I don't think we had Binder tests for SB. It also allowed me to debug through it all.